components/ProblemEditor/MetadataForm/MetadataForm.tsx

```diff
@@ -179,6 +179,39 @@ export const MetadataForm = (props: Props) => {
                 </MenuItem>
               </Select>
             </FormControl>
+            <FormControlLabel
+              control={
+                <Checkbox
+                  name="Is Short"
```

Comment:
change naming convention for name property to camel case for consistency. e.g. `name="isShort";`

repeat for all three.